opensearch_dashboards.json

```diff
@@ -6,7 +6,8 @@
   "ui": true,
   "requiredBundles": [],
   "requiredPlugins": [
-    "navigation"
+    "navigation",
+    "opensearchDashboardsUtils"
```

Comment:
needed for `createGetterSetter()` used in `services.ts`. Was recently removed due to it causing build errors, but without it it causes runtime errors. Correct pattern is to include this in requiredPlugins.